addons/mail/static/tests/new/composer/composer_tests.js

```diff
@@ -954,3 +954,20 @@ QUnit.test(
         assert.containsNone(target, ".o-mail-attachment-list .o-mail-attachment-card");
     }
 );
+
+QUnit.test("Show a thread name in the recipient status text.", async function (assert) {
+    const pyEnv = await startServer();
+    const resPartnerId1 = pyEnv["res.partner"].create({ name: "test name" });
+    const { click, openView } = await start();
+    await openView({
+        res_model: "res.partner",
+        res_id: resPartnerId1,
+        views: [[false, "form"]],
+    });
+    await click(".o-mail-chatter-topbar-send-message-button");
+    assert.strictEqual(
+        document.querySelector(".o-mail-followerTo").textContent.replace(/\s+/g, ""),
+        'To:Followersof"testname"',
+        "basic rendering when sending a message to the followers and thread does have a name"
```

Comment:
Let's remove most of assertion comments. They are redundant with assertion, so it's just additional noise.